### PR TITLE
refactor: merge CLI repository

### PR DESCRIPTION
**Pull request by venjiang**
Merge CLI repository

### CODE REVIEW

File: example/7-wasm/sfn/zig/src/zig-cache/h/41103d45e509f767566bed729b8bfafd.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+894 44871461 1664271716000000000 963580af35b2a51b2228d97caf996215 /Users/venjiang/gopath/src/github.com/yomorun/yomo/example/7-wasm/sfn/zig/src/main.zig
```
Comment (wujunzhuo) [inline example/7-wasm/sfn/zig/src/zig-cache/h/41103d45e509f767566bed729b8bfafd.txt:1]: zig-cache需要吗？

Comment (venjiang) [reply example/7-wasm/sfn/zig/src/zig-cache/h/41103d45e509f767566bed729b8bfafd.txt:1]: my fault.

**Review by fanweixiao — APPROVED** [review]

**Comment by fanweixiao** [comment]
@venjiang I remove two `.wasm` files from git repo, please review.

**Comment by venjiang** [comment]
> @venjiang I remove two `.wasm` files from git repo, please review.

`example/7-wasm/sfn/sfn.wasm` should be kept for easy running of the sample, it is only 43k size.

**Review by fanweixiao — APPROVED** [review]

**Pull request merged**